installing/installing-mirroring-installation-images.adoc

```diff
@@ -70,6 +70,48 @@ In production environments, add the required images to a registry in your restri
  modules/installation-performing-disconnected-mirror-without-registry.adoc[leveloffset=+2]
 ////
 
+[id="mirror-registry"]
+== Mirror registry for {product-title}
+
+This section introduces how to install mirror registry for {product-title}, a tool that holds mirrors of {product-title} images.
+
+Mirror registry allows users to install a small-scale version of Red Hat Quay and its required components using the `mirror-registry` command line interface (CLI) tool. Mirror registry comes equipped with pre-configured local storage and a local database. It also includes pre-configured user credentials and access permissions with a single set of inputs and no additional configuration choices to get started.
+
+Mirror registry provides a pre-determined network configuration and reports deployed component credentials and URIs upon success. A limited set of optional configuration inputs like fully qualified domain name (FQDN) services, superuser name and password, and custom TLS certificates are also provided. These services allow users to easily create an offline mirror of all {product-title} release content when running {product-title} in restricted network environments.
+
+Mirror registry is limited to hosting images that are required to install a disconnected {product-title} cluster. It uses local storage on your Red Hat Enterprise Linux (RHEL) machine, and storage supported by RHEL is supported by mirror registry. Mirror registry is not supported for production load, and is intended for use with {product-title} images only.
+
+Downloading mirror registry includes the following images:
+
+* Quay: registry.redhat.io/quay/quay-rhel8:v3.6.1
+* Redis: registry.redhat.io/rhel8/redis-6:1-25
+* Postgres: registry.redhat.io/rhel8/postgresql-10:1-161
+* Mirror registry: quay.io/quay/mirror-registry-ee:latest
```

Comment:
Lines 77-89 is a pretty large chunk of info that could probably be put into a conceptual module.